AssignmentA/AssignmentA/IFraudConditions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AssignmentA
+{
+    interface IFraudConditions
+    {
+        bool NotOwner(string _travellerName, string _nameOnCard);
```

Comment:
Local variables should NOT be prefixed with an _ (underscore).
Eg. `string travelerName`